src/components/activity/screens/ActivityAdd/containers/ActivityAddContainer.js

```diff
@@ -143,8 +162,9 @@ function mapStateToProps(state) {
 
 function mapDispatchToProps(dispatch) {
   return {
-    addActivity: (activity) => dispatch(addActivity(activity))
-  }
+    addActivity: (activity) => dispatch(addActivity(activity)),
+    addInterlocutor: (interlocutor) => dispatch(addInterlocutor(interlocutor))
+  };
```

Comment:
Why addInterlocutor is here ? Should not be in that PR ...

Reply:
Fixed